.docker/Dockerfile

```diff
@@ -13,45 +13,50 @@ RUN apt-get update && \
       gcc \
       git \
       intltool \
-      libglib2.0-dev \
-      libxml2-dev \
-      meson \
+      python3-pip \
+      ninja-build \
       pkg-config \
       unzip \
       wget \
+      libxml2-dev libglib2.0-dev cmake libusb-1.0-0-dev gobject-introspection \
+                 libgtk-3-dev gtk-doc-tools  xsltproc libgstreamer1.0-dev \
+                 libgstreamer-plugins-base1.0-dev libgstreamer-plugins-good1.0-dev \
+                 libgirepository1.0-dev gettext \
     && rm -rf /var/lib/apt/lists/*
 
-RUN wget http://ftp.acc.umu.se/pub/GNOME/sources/aravis/0.6/aravis-0.6.4.tar.xz && \
-    tar xfJ aravis-0.6.4.tar.xz && \
-    rm aravis-0.6.4.tar.xz && \
-    cd aravis-0.6.4 && \
-    ./configure && \
-    make && \
-    checkinstall --pkgname aravis --requires="libglib2.0-dev, libxml2-dev" && \
-    ldconfig
+RUN pip3 install meson
+
+RUN wget https://github.com/AravisProject/aravis/releases/download/0.8.30/aravis-0.8.30.tar.xz -O aravis-0.8.30.tar.xz &&\
+    tar xfJ aravis-0.8.30.tar.xz && \
+    rm aravis-0.8.30.tar.xz && \
+    cd aravis-0.8.30 && \
+    meson setup build && \
+    cd build && \
+    ninja && \
+    checkinstall --pkgname aravis --pkgversion 0.8.30 ninja install
 
 SHELL ["/bin/bash", "-c"]
-RUN wget https://software.micro-epsilon.com/scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip && \
-    unzip scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip && \
+RUN wget https://software.micro-epsilon.com/scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip -O scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip && \
+    unzip scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip -d scanCONTROL-Linux-SDK/ && \
     rm scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip && \
-    cd /scanCONTROL\ Linux\ SDK\ ${SCANCONTROL_SDK_VERSION}/libmescan/ && \
+    cd scanCONTROL-Linux-SDK/libmescan/ && \ 
     meson builddir && \
     cd builddir && \
     ninja && \
     checkinstall --pkgname mescan --pkgversion ${SCANCONTROL_SDK_VERSION} --requires="aravis \(\>= 0.6.0\)" ninja install && \
```

Comment:
Does the aravis min version need an update?

Reply:
Fixed